dags.yaml

```diff
@@ -419,3 +419,19 @@ bqetl_adm_engagements_daily:
     email: ['akomar@mozilla.com']
     retries: 2
     retry_delay: 30m
+
+bqetl_desktop_funnel:
+  description: >
+    This DAG schedules desktop funnel queries used to power the
+    [Numbers that Matter dashboard](https://protosaur.dev/numbers-that-matter/)
+  schedule_interval: 0 3 * * *
```

Comment:
We should probably move this to 04:00 since it's a use-case-specific DAG that doesn't have downstream dependencies.